src/ref/builtin.md

```diff
@@ -1393,6 +1393,26 @@ input elements, the entire input is retained. Examples:
 Etymology: Haskell.
 
 
+## tilde-abbr
+
+```elvish
+tilde-abbr $pwd
+```
+
+Replace occurrences of the user's home directory in its argument with
+a tilde (`~`). Examples:
```

Comment:
This description makes it sound like that it does a simple text replacement, but it is not the case:

1. It only replaces occurrences at the beginning of the argument.
2. It respects path boundaries.

I would prefer the following wording:

If `$path` represents a path under the home directory, replace the home directory with `~`.